site/about/glossary/_ai.qmd

```diff
@@ -0,0 +1,28 @@
+See [IBM's series on artificial intelligence](https://www.ibm.com/think/artificial-intelligence) for more in-depth resources.
+
+artificial intelligence (AI)
+: Srtificial intelligence is a broad term used to classify machines that mimic human intelligence and human cognitive functions like problem-solving and learning. 
+
+deep-learning
+: A subset of machine learning that uses multi-layered neural networks (deep neural networks) to simulate the complex decision-making power of the human brain. 
+
+generative AI
```

Comment:
```suggestion
generative AI (GenAI)
```

I see the abbreviated form almost more often than the full term, so suggesting we add it here.